modules/rosa-accessing-your-cluster-quick.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// getting_started_rosa/creating-first-rosa-cluster.adoc
```

Comment:
I believe this should be `getting_started_rosa/accessing-rosa-cluster.adoc`

Reply:
Done.